shared/components/SpeakersList/index.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable react/no-danger */
+
+import React, { PropTypes } from 'react';
+
+export const speakerType = PropTypes.shape({
```

Comment:
Possible to move this to our prop-types folder as it seems as though this would be shared across both our community/conference site